.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I thought we don't want to commit `.vscode` contents?

Reply:
This has been removed